trailblazer/store/models.py

```diff
@@ -155,8 +155,8 @@ class Delivery(Model):
 
     id = Column(types.Uuid, primary_key=True, default=uuid.uuid4)
     analysis_id = Column(ForeignKey(Analysis.id, ondelete="CASCADE"), nullable=False)
-    delivered_by = Column(ForeignKey(User.id), nullable=False)
-    delivered_date = Column(types.Date, nullable=False)
+    delivered_by = Column(ForeignKey(User.id))
+    delivered_date = Column(types.Date)
```

Comment:
Would we want to have the columns marked as non-nullable here still? So that no analyses can be added via Python with these values set to null? Feels bad but sort of good?

Reply:
Oh sorry! Meant that no Delivery entry can be added with these values set to null. Analyses are not dependent on this table having any entries. @ChrOertlin Also, this is for patching old analyses. Production would want to see analyses which are not delivered, meaning that old analyses should be set to delivered. But there it seemed like a lot of effort to fetch by whom and when it was delivered.

Reply:
But the only endpoints for adding an entry will automatically fill in the values.